lib/peep/event_handler.ex

```diff
@@ -28,7 +30,7 @@ defmodule Peep.EventHandler do
   end
 
   def detach(handler_ids) do
-    Enum.each(handler_ids, fn id -> :telemetry.detach(id) end)
+    for id <- handler_ids, do: :telemetry.detach(id)
```

Comment:
```suggestion
    for id <- handler_ids, do: :telemetry.detach(id)
    :ok
```

So the compiler knows to not build a list.